packages/common/src/types/gethGenesis.ts

```diff
@@ -0,0 +1,84 @@
+/**
+ * Interface for Geth Genesis Config
+ */
+export interface GethGenesisConfig {
+  chainId: number
+  homesteadBlock?: number
+  daoForkBlock?: number
+  daoForkSupport?: boolean
+  eip150Block?: number
+  eip150Hash?: string
+  eip155Block?: number
+  eip158Block?: number
+  byzantiumBlock?: number
+  constantinopleBlock?: number
+  petersburgBlock?: number
+  istanbulBlock?: number
+  muirGlacierBlock?: number
+  berlinBlock?: number
+  londonBlock?: number
+  mergeForkBlock?: number
+  cancunBlock?: number
+  shanghaiTime?: number
+  cancunTime?: number
+  terminalTotalDifficulty?: number
+  terminalTotalDifficultyPassed?: boolean
+  ethash?: {}
+  clique?: {
+    period?: number
+    epoch?: number
+    blockperiodseconds?: number
+    epochlength?: number
+  }
+  trustedCheckpoint?: {
+    sectionIndex: number
+    sectionHead: string
+    chtRoot: string
+    bloomRoot: string
+  }
+  trustedCheckpointOracle?: {
+    address: string
+    signers: string[]
+    threshold: number
+  }
+}
+
+/**
+ * Interface for account allocation in Geth Genesis
+ */
+export interface GethGenesisAlloc {
+  [address: string]: {
+    balance?: string
+    code?: string
+    storage?: { [key: string]: string }
+    nonce?: string
+  }
+}
+
+/**
+ * Interface for Geth Genesis object
+ */
+export interface GethGenesis {
+  config: GethGenesisConfig
+  nonce?: string
+  timestamp?: string
+  extraData?: string
+  gasLimit: string
+  difficulty?: string
+  mixHash?: string
+  coinbase?: string
+  alloc?: GethGenesisAlloc
```

Comment:
In Geth, this field appears to be mandatory/required: https://github.com/ethereum/go-ethereum/blob/f0cdc40cebd3fcb26d21ced0f1093efd7a2c949f/core/genesis.go#L66

Reply:
I had it as optional since there was some testdata that did not contain it, but it was the `invalidSpuriousDragonGethGenesis` file which is voluntarily invalid. Adjusted to marking is as required